content/CORE/Solutions/Integrations/Nextcloud.md

```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
-If the catalog doesn't load:
-* Go to **Network > Global Configuration** and confirm the **Default Gateway** and **DNS Servers** addresses are correct.
-* Open the **Shell** and `ping` an Internet address. The output confirms the system is connected to the Internet.
-{{< /expand >}}
-
-By default, the iXsystems-supported applications are shown.
-
-### Basic Install 
+![NextcloudInstall](/images/CORE/12.0/SolutionsIntegrationsNextcloudInstall.png "Nextcloud Install")
 
-{{< hint info >}}
-**Official Applications**
- 
-Official containers are pre-configured to only require a name during deployment.
-{{< /hint >}}
+Type a **Jail Name**, then disable the **NAT** checkbox and enter an available IP in the **IPv4 Address** field. 
+Select an **IPv4 Netmask** (iX recommends 24), then click **SAVE**.
 
-Go to **Apps** and click the *INSTALL* button.  Enter an *Application Name* and click *Next*.
+![NextcloudStatic1](/images/CORE/NextcloudStatic1.png "Nextcloud Static Setup")
 
-![SCALENextcloudInstall](/images/SCALE/SCALENextcloudInstall.png "SCALE Nextcloud Install")
+Once Nextcloud installs, you must add your Nextcloud IP to your Nextcloud jail trusted domains.
 
-For *Nextcloud Configuration* the *Username* defaults to *admin* and *Password* defaults to *changeme*.  It is recommended that you change both of these fields to entries specific to you as the user.  Ensure you employ a password that is both strong and can be remembered.
+Go to **Jails** and expand your Nextcloud jail, then click **> SHELL**.
 
-![SCALENextcloudConfiguration](/images/SCALE/SCALENextcloudConfiguration.png "SCALE Nextcloud Configuration")
+Enter `ee /usr/local/www/nextcloud/config/config.php` to edit your Nextcloud config file.
 
-For *Storage*, *Scaling/Upgrading Policy* and *Advanced DNS Settings* the defaults can be used for basic installation by clicking *Next* or bypassing and going directly to *Confirm Options*.  To *Confirm Options*, review the information presented and ensure that it is all correct.  Click *Submit*.
+Scroll to the `trusted_domains` section and type your Nextcloud IP as a new line item. Use the image below for reference.
 
-![SCALENextcloudConfirmOptions](/images/SCALE/SCALENextcloudConfirmOptions.png "SCALE Nextcloud Confirm Options")
+![NextcloudTrustedDomain](/images/CORE/NextcloudTrustedDomain.png "Nextcloud Add Trusted Domain")
 
-The Nextcloud application will now be present on the *Installed Applications* page.  Select *Portal* and you will be directed to the *Nextcloud* login page within your browser.
+Type <kdb>CTRL+C</kdb> to close the editor, then type **exit** to close the config file.
 
-![SCALENextcloudLogin](/images/SCALE/SCALENextcloudLogin.png "SCALE Nextcloud Login")
+Go back to **Plugins** and expand your Nextcloud instance. 
+Click **POST INSTALL NOTES** to obtain your *Nextcloud Admin User* and *Nextcloud Admin Password* information. Click **MANAGE** to access the Nextcloud login page within your browser.
```

Comment:
Same as above, if not specifically talking about the UI field and only the value items for the UI field, no emphasis required
```suggestion
Click **POST INSTALL NOTES** to obtain your Nextcloud admin user and Nextcloud admin password information. Click **MANAGE** to access the Nextcloud login page within your browser.
```